README.md

```diff
@@ -1,3 +1,5 @@
+![Build status](https://ci.appveyor.com/api/projects/status/1fbl5mjfwfmjj4vs/branch/master?svg=true)
```

Comment:
You should consider to remove this line, because it shows the build status of the master branch  in my forked repository.

Reply:
Hmm. I was never aware of it. Remove it. Currently we does not use it.